core/src/workflows/hold_entry_update.rs

```diff
@@ -39,12 +57,34 @@ pub async fn hold_update_workflow<'a>(
         Some(link.clone()),
         validation_data,
         &context
-    ))?;
+    ))
+    .map_err(|err| {
+        if let ValidationError::UnresolvedDependencies(dependencies) = &err {
+            context.log(format!("debug/workflow/hold_update: Entry update could not be validated due to unresolved dependencies and will be tried later. List of missing dependencies: {:?}", dependencies));
+            add_pending_validation(
+                entry_with_header.to_owned(),
+                dependencies.clone(),
```

Comment:
I think we could also DRY out this pending validation operation

Reply:
What hinders just putting it in a function is that the `ValidationWorkflow` specilization as well as the log messages depend on the workflow this is in and are different for each...